api/apps/articles/components/instant_articles/index.jade

```diff
@@ -0,0 +1,76 @@
+//- Markup for the individual Instant Article
+
+include mixins
+
+html(lang="en" prefix="op: http://media.facebook.com/op#")
```

Comment:
👍  at first it seemed a little weird that we're rendering jade templates from the API app, but I think it makes total sense for the use case.